File: fixtures/scale/csharp/medium/Repl/AngleCommand.cs
using Calc.Eval;

namespace Calc.Repl
{
    public static class AngleCommand
    {
        public static string Run(ReplContext context, string[] parts)
        {
            if (parts.Length < 2)
            {
                return "angle mode is " + context.Settings.Angle;
            }
            context.Settings.Angle = parts[1] == "degrees" ? AngleMode.Degrees : AngleMode.Radians;
            return "angle mode set to " + context.Settings.Angle;
        }
    }
}

File: fixtures/scale/php/medium/src/Repl/AngleCommand.php
<?php

namespace Calc\Repl;

use Calc\Eval\AngleMode;

class AngleCommand
{
    public static function run(ReplContext $context, array $parts): string
    {
        if (count($parts) < 2) {
            return 'angle mode is ' . $context->settings->angle->name;
        }
        $context->settings->angle = $parts[1] === 'degrees' ? AngleMode::Degrees : AngleMode::Radians;
        return 'angle mode set to ' . $context->settings->angle->name;
    }
}
